app/main.go

```diff
@@ -3,26 +3,14 @@ package main
 import (
 	"html/template"
 	"net/http"
-
-	"github.com/prometheus/client_golang/prometheus"
-	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
 type response struct {
 	count int
 }
 
-var (
-	dummyCounter = prometheus.NewCounter(
-		prometheus.CounterOpts{
-			Name: "requests",
-		},
-	)
-)
-
 func handler(w http.ResponseWriter, r *http.Request) {
 	data := response{count: 1}
-	dummyCounter.Inc()
 	t, err := template.ParseFiles("./templates/withResponse.html")
 	if err != nil {
 		http.Error(w, err.Error(), http.StatusBadRequest)
```

Comment:
This should not be 400 Bad request, because it is not a client error. StatusInternalServerError is better code, I think